go/base/management/commands/go_list_opt_outs.py

```diff
@@ -1,37 +1,22 @@
 from optparse import make_option
 
-from django.core.management.base import BaseCommand
-
-from go.base.utils import vumi_api_for_user
-from go.base.command_utils import get_user_by_email
+from go.base.command_utils import BaseGoAccountCommand
 from go.vumitools.opt_out import OptOutStore
 
 
-class Command(BaseCommand):
+class Command(BaseGoCommand):
```

Comment:
We import `BaseGoAccountCommand`, but use `BaseGoAccount`. Apparently we have no tests for this command.

Probably best to revert the changes to this module and create an issue to write tests and fix it separately.